tsdb/chunkenc/histo.go

```diff
@@ -246,6 +246,7 @@ func (a *HistoAppender) Append(int64, float64) {}
 // * the schema has changed
 // * the zerobucket threshold has changed
 // * any buckets disappeared
+// * there was a counter reset for an existing bucket (including Zero bucket)
```

Comment:
Probably should mention that the reset can happen in the count, too. Suggestion:

* there was a counter reset in the count of observations or in any bucket, including the zero bucket